site/content/docs/0.0/helpers/clearfix.md

```diff
@@ -4,10 +4,37 @@ title: Clearfix
 description: Quickly and easily clear floated content within a container by adding a clearfix utility.
 group: helpers
 aliases:
-  - "/helpers/"
-  - "/docs/helpers/"
-  - "/docs/0.0/helpers/"
-  - "/docs/helpers/clearfix/"
+- "/helpers/"
+- "/docs/helpers/"
+- "/docs/0.0/helpers/"
+- "/docs/helpers/clearfix/"
 ---
 
-{{< callout-soon "helper" >}}
+Easily clear `float`s by adding `.clearfix` **to the parent element**. Can also be used as a mixin.
+
+Use in HTML:
+
+```html
+<div class="clearfix">...</div>
+```
+
+The mixin source code:
+
+{{< scss-docs name="clearfix" file="scss/mixins/_clearfix.scss" >}}
+
+Use the mixin in SCSS:
+
+```scss
+.element {
+  @include clearfix;
+}
+```
+
+The following example shows how the clearfix can be used. Without the clearfix the wrapping div would not span around the buttons which would cause a broken layout.
```

Comment:
From here until `{{< /example >}}`, we'll need to temporarily comment it since we don't have yet "developed" the buttons (they're not in the documentation yet).

Reply:
yes I asked myself about that but sounds pretty logical to comment it for now

Reply:
I also added a task in #2589 to remember to uncomment it later